homeassistant/components/webostv/config_flow.py

```diff
@@ -60,9 +63,9 @@ async def async_step_import(self, import_info):
         _LOGGER.debug("WebOS Smart TV host %s imported from YAML config", self._host)
         return self.async_create_entry(title=self._name, data=data)
 
-    async def async_step_user(self, user_input=None):
+    async def async_step_user(self, user_input: ConfigType | None = None) -> FlowResult:
```

Comment:
We don't use `ConfigType` here. This is how we type this case:
https://github.com/home-assistant/core/blob/a1e7459dc52e763f9575b46369993ef5b8da7ec3/script/scaffold/templates/config_flow/integration/config_flow.py#L75-L78